app/js/components/App.jsx

```diff
@@ -7,7 +7,7 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React, { Component } from 'react';
-import HomePage from '../components/homePage/HomePage';
+import ManageApps from '../components/manageApps/ManageApps.jsx';
 import Header from '../components/common/Header';
```

Comment:
You may want to add a file extension for this line.

Reply:
The file is a .js file so there's no need to add the extension. I added the extensions for all .jsx files

Reply:
The header is a .jsx component, I understand you didn’t create it, but let us be consistent in our naming, we can’t call some component .js and some .jsx. We need to be consistent whichever one we use. Thank you.

Reply:
What I've done is to change the file into a `.jsx` and then change the import. That will do.